### PR TITLE
minor typos / casing

### PR DESCRIPTION
**Pull request by nchammas**
PyPy (not pypy): http://pypy.org/features.html
OS X (not OSX): https://www.apple.com/osx/

### CODE REVIEW

File: docs/supported.rst [hunk 1]
```diff
@@ -58,8 +58,8 @@ In terms of what's actually *known* to work:
 
 Coverage works out of the box with Hypothesis (and Hypothesis has 100% branch
 coverage in its own tests). However you should probably not use Coverage, Hypothesis
-and pypy together. Because Hypothesis does quite a lot of CPU heavy work compared
-to normal tests it really exacerbates 
+and PyPy together. Because Hypothesis does quite a lot of CPU heavy work compared
+to normal tests, it really exacerbates...
```
Comment (nchammas) [inline docs/supported.rst]: This looks like an unfinished sentence. @DRMacIver?

Comment (DRMacIver) [inline docs/supported.rst]: It does rather. Thanks, I'll fix it.

**Comment by nchammas** [comment]
Also, FYI @DRMacIver on this same file:

https://appveyor.com/ doesn't open, unfortunately. You need http://www.appveyor.com/

(I was about to roll it into this PR since it's minor, but you beat me to it.)

**Comment by DRMacIver** [comment]
Ack. Thanks. Will fix.

Thanks for all the proofreading!

**Comment by nchammas** [comment]
My pleasure! More incoming as I find them... :)

**Pull request merged**